### PR TITLE
BZ-2025586:Documentation update to reflect changes for capabilities getting dropped when using keyword  in the container's securityContext

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=2025586

Add `ALL` to the `requiredDropCapabilities` and few other small edits.

**DO NOT MERGE TO 4.6 AND 4.7 UNTIL https://github.com/cri-o/cri-o/pull/5466 MERGES**

Previews
[Security context constraints settings](https://deploy-preview-39124--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints#scc-settings_configuring-internal-oauth) -- Last paragraph in section and note.
[Example security context constraints](https://deploy-preview-39124--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints#security-context-constraints-example_configuring-internal-oauth) -- Expanded callout 5.
[Creating security context constraints](https://deploy-preview-39124--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing-security-context-constraints#security-context-constraints-creating_configuring-internal-oauth) -- Text changes and note in step 1.

### CODE REVIEW

**Comment by mburke5678** [comment]
@haircommander Is this a PR you can review? Thank you in advance.

**Comment by haircommander** [comment]
LGTM, correct that once @QiWang19 and my PRs merge this will work in 4.6 and 4.7

**Comment by mburke5678** [comment]
@sunilcio Can you please take a look. Thank you!!

File: modules/nodes-pods-autoscaling-about.adoc
Comment (mjpytlak) [inline modules/nodes-pods-autoscaling-about.adoc:68]:
```suggestion
The following example shows autoscaling for the `image-registry` `Deployment` object. The initial deployment requires 3 pods. The HPA object increases the minimum to 5 and will increase the pods up to 7, if CPU usage on the pods reaches 75%:
```

File: modules/nodes-pods-autoscaling-creating-cpu.adoc
Comment (mjpytlak) [inline modules/nodes-pods-autoscaling-creating-cpu.adoc:13]:
```suggestion
It is recommended to to use a `Deployment` or `ReplicaSet` object, unless you need a 
```

File: modules/nodes-pods-autoscaling-creating-cpu.adoc
Comment (mjpytlak) [inline modules/nodes-pods-autoscaling-creating-cpu.adoc:80]:
```suggestion
For example, the following command shows autoscaling for the `image-registry` `Deployment` object. The initial deployment requires 3 pods. The HPA object increases the minimum to 5 and will increase the pods up to 7, if CPU usage on the pods reaches 75%:
```

File: modules/nodes-pods-autoscaling-creating-memory.adoc
Comment (mjpytlak) [inline modules/nodes-pods-autoscaling-creating-memory.adoc:7]:
```suggestion
[id="nodes-pods-autoscaling-creating-memory_{context}"]
= Creating a horizontal pod autoscaler object for memory utilization by using the CLI
```

Comment (mjpytlak) [reply modules/nodes-pods-autoscaling-creating-memory.adoc:7]: The extra space between the anchor ID might continue to work, but I suggest removing the extra space while you are in this file.

File: modules/security-context-constraints-about.adoc
Comment (mjpytlak) [inline modules/security-context-constraints-about.adoc:166]:
```suggestion
The containers use the capabilities from this default list, but pod manifest authors can alter the list by requesting additional capabilities or removing some of the default behaviors. Use the `allowedCapabilities`, `defaultAddCapabilities`, and `requiredDropCapabilities` parameters to control such requests from the pods and to specify which capabilities can be requested, which ones must be added to each container, and which ones must be forbidden, or dropped, from each container.
```

Comment (mjpytlak) [reply modules/security-context-constraints-about.adoc:166]: I think `specify` might L10N easier.

**Review by mjpytlak — CHANGES_REQUESTED** [review]

**Comment by mjpytlak** [comment]
Please squash the commits ahead of merge. Thanks!

File: nodes/pods/nodes-pods-autoscaling.adoc
Comment (jeana-redhat) [inline nodes/pods/nodes-pods-autoscaling.adoc:20]: Pretty sure "not" is meant here.
```suggestion
It is recommended not to use a `Deployment` or `ReplicaSet` object unless you need a 
specific feature or behavior provided by other objects. For more information on
these objects, 
see xref:../../applications/deployments/what-deployments-are.adoc#what-deployments-are[Understanding Deployment and DeploymentConfig objects].
```

File: modules/nodes-pods-autoscaling-creating-memory.adoc
Comment (jeana-redhat) [inline modules/nodes-pods-autoscaling-creating-memory.adoc:16]:
```suggestion
It is recommended to not use a `Deployment` or `ReplicaSet` object unless you need a 
```
(same as the other instance of this note)

File: modules/nodes-pods-autoscaling-creating-memory.adoc
Comment (jeana-redhat) [inline modules/nodes-pods-autoscaling-creating-memory.adoc:111]: I have no idea if this is correct or not, but maybe consider:
```suggestion
<4> Specify the type of object. The object must be a `Deployment`, `DeploymentConfig` (`dc`), 
`ReplicaSet` (`rs`), `ReplicationController` (`rc`), or `StatefulSet`.
```

File: modules/nodes-pods-autoscaling-creating-memory.adoc
Comment (jeana-redhat) [inline modules/nodes-pods-autoscaling-creating-memory.adoc:163]: Same comment as above about () vs /

File: modules/nodes-pods-autoscaling-creating-cpu.adoc
Comment (jeana-redhat) [reply modules/nodes-pods-autoscaling-creating-cpu.adoc:13]: Also I think "recommended to to use" is intended as "recommended not to use" as elsewhere

File: modules/nodes-pods-autoscaling-about.adoc
Comment (jeana-redhat) [reply modules/nodes-pods-autoscaling-about.adoc:68]: Prehaps maybe even split that up a bit more:
```suggestion
The following example shows autoscaling for the `image-registry` `Deployment` object. The initial deployment requires 3 pods. The HPA object increases the minimum to 5. If CPU usage on the pods reaches 75%, the pods will increase to 7:
```

File: modules/nodes-pods-autoscaling-creating-cpu.adoc
Comment (jeana-redhat) [reply modules/nodes-pods-autoscaling-creating-cpu.adoc:80]: Prehaps maybe even split that up a bit more:
```suggestion
For example, the following command shows autoscaling for the `image-registry` `Deployment` object. The initial deployment requires 3 pods. The HPA object increases the minimum to 5. If CPU usage on the pods reaches 75%, the pods will increase to 7:
```

**Review by jeana-redhat — COMMENTED** [review]
Just a couple small additional thoughts from me. Looks good overall!

File: nodes/pods/nodes-pods-autoscaling.adoc
Comment (mburke5678) [reply nodes/pods/nodes-pods-autoscaling.adoc:20]: Nope. Just a bad cut and paste, likely.

File: modules/nodes-pods-autoscaling-creating-memory.adoc
Comment (mburke5678) [reply modules/nodes-pods-autoscaling-creating-memory.adoc:16]: Same

**Pull request merged**
